_posts/2018-03-01-Rust-1.24.1.md

```diff
@@ -0,0 +1,257 @@
+---
```

Comment:
I don't know if we're still planning on shipping today or waiting till tomorrow, but if we wait, the filename needs to change so that the URL is correct.

Reply:
shipping today, so this is resolved!